docs/reference-guides/data/data-core-editor.md

```diff
@@ -1582,6 +1582,10 @@ _Related_
 
 -   toggleBlockMode in core/block-editor store.
 
+### toggleBoundaryMode
```

Comment:
I'd be supportive of this preference, but I think it should be called "showBlockBoundaries"